packages/desktop-client/src/components/reports/chart-theme.ts

```diff
@@ -112,8 +221,12 @@ export const chartTheme = {
   },
 };
 
-export function getColorScale(name) {
-  const scales = {
+type ColorScales = {
+  [key: string]: string[];
+};
+
+export function getColorScale(name: string): string[] {
+  const scales: ColorScales = {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏would be better to use an inline style here if we want to be super-strict. Although personally I wouldn't add a manual type to it. Up to you.
```suggestion
  const scales: Record<string, string[]> = {
```

Reply:
Implemented it inline. Thanks for the suggestion!